codegen/syntax/ClassishBody.hack

```diff
@@ -158,25 +158,27 @@ final class ClassishBody extends Node {
 
   /**
    * @return NodeList<ConstDeclaration> | NodeList<IClassBodyDeclaration> |
+   * NodeList<ContextConstDeclaration> | NodeList<Node> |
```

Comment:
I think you can add `ContextConstDeclaration` to `IClassBodyDeclaration`. This one actually seems correct unlike the other 2.